Botusanu Darius-Gabriel/NewProject2/main.c

```diff
@@ -0,0 +1,26 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main()
+{
+    float x;
+    float t;
+    printf("Input your real value: ");
+    scanf("%f", &x);
+    if(x<-2)
+        {
+            t=x*x-7*x+4;
+            printf("f(x)= %f \n", t);
```

Comment:
Basically, you're printing the same thing in 3 different ifs.
Why not, if you're assigning `t` in every if, simply use this `printf` 1 time, after all the ifs?